README.md

```diff
@@ -25,6 +25,8 @@
 
 [Etsy](https://codeascraft.com/2012/01/23/solr-bittorrent-index-replication/). Sending multicast traffic without properly configuring switches caused an Etsy global outage.
 
+[Facebook](https://engineering.fb.com/2021/10/04/networking-traffic/outage/). October 4th 2021 global outage - Configuration changes on the backbone routers.
```

Comment:
Change summary to a complete sentence.

Reply:
updated

Reply:
this: https://engineering.fb.com/2021/10/05/networking-traffic/outage-details/

Reply:
thx